cmd/guacgql/cmd/server.go

```diff
@@ -81,9 +83,8 @@ func startServer(cmd *cobra.Command) {
 		logger.Errorf("unable to initialize graphql server: %v", err)
 		os.Exit(1)
 	}
-	// Setup Prometheus metrics handler
-	metric, err := cli.SetupPrometheus(ctx, logger, "guacgql")
```

Comment:
Why did we move this function?

Reply:
I was looking at that function a few PRs ago and when I revisited the `"github.com/guacsec/guac/pkg/cli` package in the course of this header work, I saw that there was no other caller of the function and felt it was misplaced in `cli/init.go`. So I decided to move it to where it was used and to make it private when I did. It is not relevant to the CLI command HTTP header functionality.

Reply:
Thank you for the explanation!